<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1092</width>
    <height>741</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="window">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>20</y>
     <width>681</width>
     <height>641</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QPushButton" name="open_btn">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>690</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </widget>
  <widget class="QPushButton" name="camera_btn">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>690</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save_btn">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>690</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="HSV">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>331</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>HSV</string>
   </property>
   <widget class="QSlider" name="hue_slider">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="hue">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>hue</string>
    </property>
   </widget>
   <widget class="QLabel" name="sat">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>saturation</string>
    </property>
   </widget>
   <widget class="QSlider" name="sat_slider">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="value">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>value</string>
    </property>
   </widget>
   <widget class="QSlider" name="value_slider">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>110</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="RGB">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>331</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>RGB</string>
   </property>
   <widget class="QLabel" name="green">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>green</string>
    </property>
   </widget>
   <widget class="QLabel" name="red">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>red</string>
    </property>
   </widget>
   <widget class="QLabel" name="blue">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>blue</string>
    </property>
   </widget>
   <widget class="QSlider" name="red_slider">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="green_slider">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>100</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="blue_slider">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="Filter">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>331</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>Filtering</string>
   </property>
   <widget class="QSlider" name="light_slider">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="light">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>light</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cartoon_btn">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>120</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>cartoon</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sketch_btn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>sketch</string>
    </property>
   </widget>
   <widget class="QLabel" name="filter">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>filter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sharpning_btn">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>sharpning</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bilateral_btn">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>80</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>bilateral</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="Drawing">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>550</y>
     <width>331</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Drawing</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rec_btn">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>680</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>REC ON</string>
   </property>
  </widget>
  <widget class="QPushButton" name="capture_btn">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>710</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Capture</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
